core/src/tile/tile.h

```diff
@@ -51,7 +49,7 @@ class Tile {
     float getScale() const { return m_scale; }
 
     /* Returns the reciprocal of <getScale()> */
-    float getInverseScale() const { return m_inverseScale; }
+    float getInverseScale() const { return 1.0f / m_scale; }
```

Comment:
I believe the distinction is fairly arbitrary, it would be reasonable to reconsider this member now that we have a broader view of the tile pipeline.